### PR TITLE
Logs to blob

### PR DESCRIPTION
**Pull request by constantinpr**
### JIRA link ###

https://tools.hmcts.net/jira/browse/RDO-2419

### Change description ###

Enabling logs to blob to our webapps across all environments  

Succesful builds in our sandbox environment:

https://sandbox-build.platform.hmcts.net/job/HMCTS_Sandbox_CNP/job/cnp-plum-recipes-service/job/logs-to-blob/

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: templates/asp-app.json [hunk 1]
```diff
@@ -95,16 +95,101 @@
     },
     "java_container_version": {
       "type": "string"
+    },
+    "storageAccountSkuName": {
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "metadata": {
+        "description": "The name of the App Service Plan."
```
Comment (timja) [inline templates/asp-app.json]: this doesn't seem right?

File: templates/asp-app.json [hunk 2]
```diff
@@ -95,16 +95,101 @@
...
+      }
+    },
+    "storageAccountKind": {
+      "type": "string",
+      "defaultValue": "StorageV2",
+      "metadata": {
+        "description": "The name of the Storage Account Type."
```
Comment (timja) [inline templates/asp-app.json]:
```suggestion
        "description": "The name of the Storage Account Kind."
```

File: templates/asp-app.json [hunk 3]
```diff
@@ -95,16 +95,101 @@
...
+        "type": "blobServices/containers",
+        "apiVersion": "2018-02-01",
+        "dependsOn": [
+          "[concat('Microsoft.Storage/storageAccounts/', variables('storageAccountName'))]"
+        ],
+        "properties": {
+          "publicAccess": "Blob"
+        }
+      }]
+    }, {
```
Comment (timja) [inline templates/asp-app.json]:
```suggestion
    }, 
    {
```

File: templates/asp-app.json (hunk 1 above)
Comment (constantinpr) [reply templates/asp-app.json]: changed the description to : The name of replciation strategy - LRS Locally-redundant storage.

**Pull request closed without merging**